pkgs/test/pubspec.yaml

```diff
@@ -46,4 +46,4 @@ dependency_overrides:
   test_core:
     path: ../test_core
   test_api:
-    path: ../test_api
+    path: ../test_api
```

Comment:
[nit] don't drop the trailing newline

Reply:
Ah ya I need to figure out how to configure this in vscode again lol

Reply:
I have:

```
"files.trimTrailingWhitespace": true,
"files.insertFinalNewline": true,
```

 :-)